src/ussd_registration.js

```diff
@@ -0,0 +1,343 @@
+go.app = function() {
+    var vumigo = require('vumigo_v02');
+    var App = vumigo.App;
+    var Choice = vumigo.states.Choice;
+    var ChoiceState = vumigo.states.ChoiceState;
+    var PaginatedChoiceState = vumigo.states.PaginatedChoiceState;
+    var EndState = vumigo.states.EndState;
+    var FreeText = vumigo.states.FreeText;
+
+
+    var GoApp = App.extend(function(self) {
+        App.call(self, 'state_start');
+        var $ = self.$;
+        var interrupt = true;
+
+        self.init = function() {
+
+            // Load self.contact
+            return self.im.contacts
+                .for_user()
+                .then(function(user_contact) {
+                   self.contact = user_contact;
+                });
+        };
+
+
+    // TEXT CONTENT
+
+        var questions = {
+            "state_timed_out":
+                "You have an incomplete registration. Would you like to continue with this registration?",
+            "state_auth_code":
+                "Welcome to Hello Mama! Please enter your unique personnel code. For example, 12345",
+            "state_msisdn":
+                "Please enter the mobile number of the person who will receive the weekly messages.  For example 0803304899",
+            "state_msg_receiver":
+                "Please select who will receive the messages on their phone:",
+            "state_pregnancy_status":
+                "Please select one of the following:",
+            "state_last_period_month":
+                "Please select the month the woman had her last period:",
+            "state_last_period_day":
+                "What day of the month did the woman start her last period? For example, 12.",
+            "state_baby_birth_date":
+                "Select the month & year the baby was born:",
+            "state_baby_birth_day":
+                "What day of the month was the baby born? For example, 12.",
+            "state_msg_language":
+                "Which language would this person like to receive these messages in?",
+            "state_msg_call_or_text":
```

Comment:
Let's call this 'state_msg_type' for better corellation with the voice_registration app